dockerfiles/unit/Dockerfile

```diff
@@ -66,4 +66,15 @@ RUN curl -L https://releases.hashicorp.com/vault/0.7.3/vault_0.7.3_linux_amd64.z
 RUN curl -fsSL https://releases.hashicorp.com/terraform/0.12.6/terraform_0.12.6_linux_amd64.zip -o /tmp/terraform.zip && \
       unzip /tmp/terraform.zip -d /usr/local/bin && \
       rm /tmp/terraform.zip
-RUN apt-get update && apt-get -y install jq openssh-client
+RUN apt-get update && apt-get -y install jq openssh-client libnss3-tools
+RUN curl -so /etc/ssl/certs/fake_root.pem \
```

Comment:
sick but I don't know what some of these things do. but lgtm

Reply:
it's all for the sake of https://letsencrypt.org/docs/staging-environment/

Reply:
and getting chromium to trust certs requires the NSS DB stuff https://thomas-leister.de/en/how-to-import-ca-root-certificate/#browser-firefox-chromium